components/brave_wallet_ui/common/hardware/ledgerjs/filecoin_ledger_keyring.ts

```diff
@@ -55,44 +60,79 @@ export default class FilecoinLedgerKeyring implements LedgerFilecoinKeyring {
     return this.provider !== undefined
   }
 
+  makeApp = async (): Promise<Boolean> => {
+    if (this.transportWrapper) {
+      await this.transportWrapper.disconnect()
+    }
+    this.transportWrapper = new TransportWrapper()
+    await this.transportWrapper.connect()
+    let provider = new LedgerProvider({
+      transport: this.transportWrapper.transport,
+      minLedgerVersion: {
```

Comment:
Seeing this isn't changed so asking just for context. What motivated this to be the minLedgerVersion being set?

Reply:
it is required to be set by LedgerProvider

Reply:
Ok, but we don't have a minimum then so just set it to the lowest possible value?

Reply:
yes

Reply:
Awesome, thanks for the additional info